tools/serverpod_cli/test/generator/dart/client_code_generator/static_comment_test.dart

```diff
@@ -72,5 +72,16 @@ void main() {
         );
       }
     });
+
+    test(
+        'then all code contains ignore linting for invalid_use_of_internal_member',
+        () {
+      for (var code in codeMap.values) {
+        expect(
+          code,
+          contains('// ignore_for_file: invalid_use_of_internal_member'),
```

Comment:
How is this added? I can't seem to see the relevant code gen change in this PR.

Or did this get detected automatically already?

Reply:
I added this test :D

Reply:
Change is in tools/serverpod_cli/lib/src/generator/code_generator.dart

Reply:
Hmm, missed that.

Will be happy day when I get access to https://github.blog/changelog/2025-06-26-improved-pull-request-files-changed-experience-now-in-public-preview/